Documentation.md

```diff
@@ -105,8 +105,83 @@ Users have to fill in the fields during the installation process first, on the i
 
 ![Authentication admin](./assets/images/marketplace-configure.png)
 
-## App Settings
-This part is still under development. This is where you will be able to add specific configuration steps such as mapping.
+## App Settings (in progress)
+As the authentication section, you can configure setting parameters that users will need to configure.
+
+**Define app's setting fields**
+You can add fields and define the type of entry you need.
+
+* Label: it’s the name of your field, this is what users will see.
+* Type: it defines the type of entry.
+* ID: you should choose the name of your field ID according to your code.
+
+You can add as much fields as you need.
+This is the second steps of users installation on the iAdvize Marketplace.
+Fields appear to users according to their order of creation (the 1st entry created is the 1st on displayed on the page).
+
+* Configure your [verification Url](#verification-url)
+This is the URL on which we will check your app. We'll call this URL and your connector will be sent back if the setting parameters have been correctly set up to go to the next installation step.
+
+
+Users have to fill in the fields during the installation process second, on the iAdvize Marketplace.
+
+
+## Verification url
```

Comment:
Verification url

In the App Authentication and App Settings sections of the iAdvize Developer Platform, you can set a "Verification url". 

We will call this URL each time the application requires a confirmation from your connector to approve the configuration settings. 
It means that your connector must answer to this call by a confirmation or an error message to determine whether the setting parameters have been correctly set up. If the first step is confirmed by your connector, it goes to the next step of configuration.